src/java.base/share/classes/java/util/concurrent/ForkJoinWorkerThread.java

```diff
@@ -244,9 +265,20 @@ static final class InnocuousForkJoinWorkerThread extends ForkJoinWorkerThread {
         public void setUncaughtExceptionHandler(UncaughtExceptionHandler x) { }
 
         @Override // paranoically
+        @SuppressWarnings("removal")
         public void setContextClassLoader(ClassLoader cl) {
-            if (cl != null && ClassLoader.getSystemClassLoader() != cl)
+            if (System.getSecurityManager() != null &&
```

Comment:
@AlanBateman Alternatives for this? 🤔

Reply:
I don't think it needs an alternative, instead L270-272 will be removed as part of the SM cleanup.

Reply:
Sounds like a good plan 👍